ansible/roles/codeready_containers_hacks/defaults/main.yml

```diff
@@ -0,0 +1,24 @@
+---
+# defaults file for codeready-containers-hack
+crc_version: latest
+crc_url: https://mirror.openshift.com/pub/openshift-v4/clients/crc/
+crc_file_name: crc-linux-amd64.tar.xz
+pull_secret_path: /tmp/pull-secret.txt
+pull_secret_content: |
+  changeme
+use_all_in_one_haproxy: true
+haproxy_ip: ""
+use_all_in_one_dnsmasq: true 
+log_level: info
+crc_ip_address: 192.168.130.11
+delete_crc_deployment: false
+
+ocp4_release: latest
+ocp4_version: latest
+ocp4_release_url: "https://mirror.openshift.com/pub/openshift-v4/clients/ocp/{{ ocp4_release }}/"
+ocp4_client: "openshift-client-linux-{{ ocp4_version }}.tar.gz"
+remove_oc_tool: false
+# Default memory in MiB
+memory_crcvm: 9216
+# Defaul cpus
+vcpu_crcvm: 4
```

Comment:
Please prefix role variables with the role name to avoid conflicts.

Ex:
```
# Default memory in MiB
codeready_containers_hacks_memory_crcvm: 9216
# Default cpus
codeready_containers_hacks_vcpu_crcvm: 4
```